appleLohFetch.js

```diff
@@ -0,0 +1,59 @@
+var fetch = require('node-fetch');
+
+var apiKey = process.env.API_KEY;
+var apiSecret = process.env.API_SECRET;
+
+var baseUrl = 'https://api.parsely.com/v2';
```

Comment:
Recommend creating another function/file called parsleyClient, because there's some repeated code (e.g. `baseUrl` and the `fetch` function)